### ISSUE TITLE
[Docs] refine type in DebuggingDirectivesExamplesSpec.scala

### ISSUE THREAD

**Issue by danielkman** [akka/akka-http#1520]
In akka-http/docs/src/test/scala/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala line 119
The res param is typed as Any, if I understand correctly it can only be of type HttpResponse.

If you agree, I'll be happy to submit a PR to fix that

**Comment by ktoso** [comment]
It's not a response; look at the match:

```
val entry = res match {
        case Complete(resp) =>
```

It's a `RouteResult`, so it could indeed be:

```
      level:            LogLevel       = Logging.InfoLevel)(req: HttpRequest)(res: RouteResult): Unit = {
```

**Comment by ktoso** [comment]
Thanks for submitting a PR with that change in advance!

**Comment by jrudolph** [comment]
Fixed by #1523.
